presto-docs/src/main/sphinx/functions/hyperloglog.rst

```diff
@@ -88,3 +88,8 @@ Functions
 
     Returns the ``HyperLogLog`` of the aggregate union of the individual ``hll``
     HyperLogLog structures.
+
+.. function:: merge_hll(Block<HyperLogLog>) -> HyperLogLog
```

Comment:
I think this should be `array(HyperLogLog)`, `Block` is an internal datastructure unknown to our users.